src/ssl.c

```diff
@@ -201,6 +201,45 @@ byte tsip_rootCAverified( );
 #endif
 
 #ifdef WOLFSSL_SESSION_EXPORT
+/**
```

Comment:
Is it possible our doxygen scripts won't pickup this up unless it is in `doc/dox_comments/header_files/ssl.h`?